specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-02-01-preview/examples/triggeredAnalyticsRuleRuns/triggerRuleRun_Post.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2023-02-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "ruleId": "65360bb0-8986-4ade-a89d-af3cf44d28aa",
+    "AnalyticsRuleRunTriggerParameter": {
+      "properties": {
+        "executionTimeUtc": "2022-12-22T15:37:03.074Z"
+      }
+    }
+  },
+  "responses": {
+      "202": {
+        "headers": {
+          "Code": "202",
+          "Message": "Accepted",
+          "Location": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalIinsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/alertRules/65360bb0-8986-4ade-a89d-af3cf44d28aa/triggerRuleRun"
```

Comment:
the location should refer to the run with the id
i.e. end with something like "triggeredAnalyticsRuleRuns/5abbc58b-9655-4f9b-80ac-5a576753e4ec?api-version=2023-02-01-preview" instead of "triggerRuleRun"

Reply:
fixed :)